tmt/steps/provision/testcloud.py

```diff
@@ -184,6 +184,8 @@ class TestcloudGuestData(tmt.steps.provision.GuestSshData):
     image_url: Optional[str] = None
     instance_name: Optional[str] = None
 
+    _normalize_key = tmt.utils.NormalizeKeysMixin._normalize_string_list
```

Comment:
I'd put this one into a parent class - `key` is not defined here but in `GuestSshData`.

Reply:
Moved and rebased.

Reply:
Looks good to me. 

Maybe also need add the key option in schema ?

Reply:
Aah, right. Schema. Will do

Reply:
Thanks